### PR TITLE
[pacemaker] Replace deprecated "pcs property list" command

### PR DESCRIPTION
**Pull request by yinoguchi**
This patch replaces "pcs property list" with "pcs property config" because "pcs property list" is deprecated and will be removed:

```
# pcs property list --all 
Deprecation Warning: This command is deprecated and will be removed. Please use 'pcs property config' instead.
Cluster Properties:
...
```
This warning message was introduced in pcs v0.10.9:
https://github.com/ClusterLabs/pcs/commit/332f56ade4773bf22e2ee6c8c91962506b844135#diff-4c58cd856f4e8b23a9d99d961c357e224a9f6314542d50add71a04a2b0b6980cR96-R104

---
Please place an 'X' inside each '[]' to confirm you adhere to our [Contributor Guidelines](https://github.com/sosreport/sos/wiki/Contribution-Guidelines)

- [x] Is the commit message split over multiple lines and hard-wrapped at 72 characters?
- [x] Is the subject and message clear and concise?
- [x] Does the subject start with **[plugin_name]** if submitting a plugin patch or a **[section_name]** if part of the core sosreport code?
- [x] Does the commit contain a **Signed-off-by: First Lastname <email@example.com>**?
- [x] Are any related Issues or existing PRs [properly referenced](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) via a Closes (Issue) or Resolved (PR) line?

### CODE REVIEW

**Review by arif-ali — APPROVED** [review]

**Review by pmoravec — COMMENTED** [review]
There theoretically might be old versions of `pcs` deployed (i.e. RHEL8.4 ships `0.10.8` which should have `pcs property list` only).

@sbradley7777 , do you know if it could be a problem?

**Comment by jesusserranost** [comment]
@pmoravec - This will be definitely a problem unless there is some kind of internal check which runs either `pcs property list --all` or `pcs property config --all`, depending on the currently installed version of the `pcs` package. I can confirm there are still clusters deployed with older versions which are still officially supported (as of today), for which this change will break the functionality. I just tested this on a RHEL 8.2 lab and the new command just prints the "Usage help syntax", as expected:

```
[root@fastvm-rhel-8-2-201 ~]# pcs property config --all | head -5

Usage: pcs property ...
    [list|show [<property> | --all | --defaults]] | [--all | --defaults]
        List property settings (default: lists configured properties).
        If --defaults is specified will show all property defaults, if --all
```

**Comment by pmoravec** [comment]
There are a few options:
- collect both outputs (once duplicated or invalid command)
- collect one output depending on `pcs` version - we have a check for it, see https://github.com/sosreport/sos/blob/main/sos/collector/clusters/pacemaker.py#L14 and https://github.com/sosreport/sos/blob/main/sos/collector/clusters/pacemaker.py#L66 applied to something like https://github.com/sosreport/sos/blob/main/sos/report/plugins/networking.py#L270 .

I would personally go for 2nd option: just have a check and use appropriate option per it.

**Comment by yinoguchi** [comment]
Thank you for the review. 
As suggested, I have made modifications to check `pcs` version and change the command to be executed accordingly.

File: sos/report/plugins/pacemaker.py [hunk 1]
```diff
@@ -42,13 +43,21 @@ def setup_crm_shell(self):
...
+
         self.add_copy_spec("/var/log/pcsd/pcsd.log")
         self.add_cmd_output([
             "pcs stonith sbd status --full",
             "pcs stonith sbd watchdog list",
             "pcs stonith history show",
-            "pcs property list --all"
         ])
+
+        pcs_version = '.'.join(pcs_pkg['version'])
```
Comment (pmoravec) [inline sos/report/plugins/pacemaker.py:55]: Try running `sos report -o pcaemaker` on a system *without* `pcs` package installed - you get

    TypeError: 'NoneType' object is not subscriptable

here (CI tests fail on this)

Comment (yinoguchi) [reply sos/report/plugins/pacemaker.py:57]: Thanks again. How about this? I had added a return if the `pcs` package does not exist. All commands within the `setup_pcs()` depend on the `pcs` package, so I believe this should be fine. 
Of course, if there are any problems with this, I will make corrections as per your suggestion.

Comment (yinoguchi) [reply sos/report/plugins/pacemaker.py:57]: Oh, sorry.
```
flake8 sos
sos/report/plugins/pacemaker.py:48:12: E111 indentation is not a multiple of 4
```
I will fix this and push it again.

**Review by pmoravec — APPROVED** [review]
What an elegant solution I have overlooked :)

**Review by TurboTurtle — APPROVED** [review]

**Pull request merged**